src/jit/environment.cc

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2017 wasmjit-omr project participants
```

Comment:
Should be 2018 now 🙂 .

Reply:
Oops. Copied the header from another file without checking that. Should be fixed now.